RELEASE_NOTES.md

```diff
@@ -1,13 +1,8 @@
-## Enhancements  
-- [KOGITO-4245](https://issues.redhat.com/browse/KOGITO-4245) - Use Infinispan CR spec data instead of status to fetch credentials Secret 
-- [KOGITO-2317](https://issues.redhat.com/browse/KOGITO-2317) - Unify Kogito Quarkus and Spring Images
-- [KOGITO-4059](https://issues.redhat.com/browse/KOGITO-4059) - Refactor Kogito operator into internal and core component
-- [KOGITO-4173](https://issues.redhat.com/browse/KOGITO-4173) - Adjust default probe URL for Quarkus
-- [KOGITO-3784](https://issues.redhat.com/browse/KOGITO-4173) - Update CustomResourceDefinition to apiextensions.k8s.io/v1
-- [KOGITO-233](https://issues.redhat.com/browse/KOGITO-233)   - Migrate Operator image generation to CEKit 
```

Comment:
The KOGITO-4059 and KOGITO-3784 were added on master after the 1.3.x cut-off and they will be in our next upcoming release(1.4.x) which is not branched yet. Can we add them back in the release_notes?/
Also if you add them can you please also add a breaking compatibility with KOGITO-3784(The CRDs will stop working with k8s<=1.15)
Thanks

Reply:
> Also if you add them can you please also add a breaking compatibility with KOGITO-3784(The CRDs will stop working with k8s<=1.15)

This will be in the Github release notes. The `KOGITO-4059` was missing the fixVersion. We need to integrate our JIRAs with merged PRs somehow to set these fields automatically. :/